lib/ca_certs_stubs.c

```diff
@@ -0,0 +1,51 @@
+#include "caml/alloc.h"
+#include "caml/fail.h"
+#include "caml/memory.h"
+
+#ifdef _WIN32
+
+#include <windows.h>
+
+static value alloc_cons(value v_hd, value v_tl)
+{
+    CAMLparam2(v_hd, v_tl);
+
+    CAMLlocal1(v_cons);
+    v_cons = caml_alloc(2, 1);
+    Store_field(v_cons, 0, v_hd);
+    Store_field(v_cons, 1, v_tl);
+
+    CAMLreturn(v_cons);
+}
+
+value ca_certs_windows_trust_anchors_der(value v_unit)
+{
+    CAMLparam1(v_unit);
+
+    CAMLlocal2(v_list, v_encoded_cert);
+    v_list = Val_int(0);
+
+    HCERTSTORE hCertStore = CertOpenSystemStore(0, "ROOT");
```

Comment:
Check `hCertStore` here and raise if it's `NULL`?

Reply:
yes, I'll do that (+ check on close too)